### DISCUSSION TITLE
Overwrite copyright to include images

### DISCUSSION THREAD

**Discussion by sumau** [squidfunk/mkdocs-material#4893] [General]
I am trying to recreate [this](https://tdt-documentation.london.cloudapps.digital/) template using mkdocs material. I have overwritten copyright.html partial so that it includes images:

```
<div class="md-copyright">
        <style>          
            .logo {
                float: right;    
                 margin: 0 0 0 15px;
                }
        </style>

        <img class="logo" src="assets/images/govuk-crest-2x.png" width=125px height=102px; >

        <img src="assets/images/ogl.png" >
        All content is available under the 
        <a
            href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
            target="_blank" rel="noopener"
        >
            Open Government Licence v3.0</a>
        , except where otherwise stated
</div>
```

This works fine on the index but the images fail to appear on the other pages when I serve the website. Should I not be using copyright.html partial?

**Comment by sumau** [comment]
The same kind of problem happens if I overwrite the footer block in main.html:

```
{% block footer %}
  <footer class="md-footer">
    <div class="md-footer-meta__inner md-grid">

      <div class="md-copyright">
        <img src="assets/images/ogl.png" >
        All content is available under the 
        <a
            href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
            target="_blank" rel="noopener"
        >
            Open Government Licence v3.0</a>
        , except where otherwise stated
      </div>

      <div>
        <img class="logo" src="assets/images/govuk-crest-2x.png" width=125px height=102px; >
      </div>

    </div>
  </footer>
{% endblock %}
```

**Reply by squidfunk** [reply]
We can't help you without a [reproduction](https://squidfunk.github.io/mkdocs-material/bug-report/reproduction/). Everything else would be guesswork. If you provide one, I can look into it.

**Reply by sumau** [reply]
ok will send one tomorrow thanks 👍

**Reply by sumau** [reply]
I followed all the steps in [reproduction](https://squidfunk.github.io/mkdocs-material/bug-report/reproduction/) apart from `mkdocs build` which fails because I'm customizing the theme. However I have still attached the [zip file](mkdocs-reproduction-zip.zip) (excluding the venv). You can also see the problem live on [this]( https://ministryofjustice.github.io/mkdocs-tech-docs-template/) website where the images are displayed on the footer on the home page but not the Contribute page. Please let me know if there's anything other info I can provide. Appreciate your support on this since I know customisation is out of remit :-)

[mkdocs-reproduction-zip.zip](https://github.com/squidfunk/mkdocs-material/files/10474244/mkdocs-reproduction-zip.zip)

**Comment by SoumayaMauthoorMOJ** [comment]
Hello @squidfunk any udpdate on the above? As I said I know this is a customization issue which is out of remit so I completely understand if it takes longer to resolve :-)

**Reply by squidfunk** [reply]
Not yet – I have this on my plate to check, but since it's a customization, it's low priority.

**Reply by SoumayaMauthoorMOJ** [reply]
Understood thanks for the update :-)

**Comment by SoumayaMauthoorMOJ** [comment]
Hello @squidfunk all fixed now -  I had forgotten to include [base_url](https://www.mkdocs.org/dev-guide/themes/#base_url) template variable  to ensure that the link is always relative to the current page:

```
{% block footer %}
  <footer class="md-footer">
    <div class="md-footer-meta__inner md-grid">

      <div class="md-copyright">
        <img src="{{ base_url }}/assets/images/ogl.png" >
        All content is available under the 
        <a
            href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
            target="_blank" rel="noopener"
        >
            Open Government Licence v3.0</a>
        , except where otherwise stated
      </div>

      <div>
        <img class="logo" src="{{ base_url }}/assets/images/govuk-crest-2x.png" width=125px height=102px;>
        <br>
        <a class="md-copyright" 
        href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/">© Crown copyright</a>
      </div>

    </div>
  </footer>
{% endblock %}
```

**Reply by squidfunk** [reply]
Perfect! So sorry I haven't found the time yet, but glad you fixed it. Also thanks for sharing! One suggestion: it's better to use the [`url` filter](https://www.mkdocs.org/dev-guide/themes/#url), as it automatically ensures that all URLs are correct when you reference files relative to `docs_dir`:

``` html
<img class="logo" src="{{ 'assets/images/govuk-crest-2x.png' | url }}" />
```

We're also using it in our overrides:

https://github.com/squidfunk/mkdocs-material/blob/d32ca846f1ea853baee72c0f762fcf876d36d658/src/.overrides/main.html#L51

**Comment by SoumayaMauthoorMOJ** [comment]
Even better thanks :-)
